src/storage/kv/snapshot.rs

```diff
@@ -13,13 +13,19 @@ pub(crate) const FILTERS: [fn(&IndexValue) -> Result<()>; 1] = [ignore_deleted];
 /// A versioned snapshot for snapshot isolation.
 pub(crate) struct Snapshot {
     snap: VartSnapshot<VariableSizeKey, IndexValue>,
+    ts: u64,
```

Comment:
Can we name this `version`?

Reply:
It is actually the timestamp. Because the version is set on the snapshot and internally it remains static, and what we are passing here is the ts as seen [here](https://github.com/surrealdb/vart/blob/2f33a89d7d890602c6f2a4d2c23f4a7199cb168a/src/snapshot.rs#L24)

Reply:
Actually let me get rid of that altogether and make every insert inside the snapshot have a ts of 0 as it hardly matters

Reply:
I've removed this, rather adding now as ts for every insert in snapshot